.github/renovate.json

```diff
@@ -10,7 +10,8 @@
     "Google.Api.Gax.Grpc.GrpcCore/**",
     "Google.Api.Gax.Grpc.GrpcNetClient/**",
     "Google.Api.Gax.Grpc.Testing/**",
-    "Google.Api.Gax.Grpc.Rest/**"    
+    "Google.Api.Gax.Grpc.Rest/**",
```

Comment:
Could you remove the old GrpcCore and GrpcNetClient lines while we're here? (Those packages no longer exist.)

Reply:
Yep, done!